crates/edr_eth/src/log/block.rs

```diff
@@ -24,15 +24,21 @@ pub struct FullBlockLog {
     #[cfg_attr(feature = "serde", serde(flatten))]
     pub inner: ReceiptLog,
     /// block hash
+    // TODO review: this is optional in Hardhat. Should we keep it as is or make it optional to
```

Comment:
All of those are either always present or none of them are present. That's why we have a `ReceiptLog` and a `FullBlockLog`. In the former case, use `FullBlockLog`. In the latter case, use `ReceiptLog`

Reply:
Got it thanks!